README_en-US.md

```diff
@@ -0,0 +1,60 @@
+# Rusbé
+
+![GitHub package.json version](https://img.shields.io/github/package-json/v/erick2280/rusbe-web-client)
+
+Rusbé is a follow up platform of the University Restaurant of UFPE.
+
+## Setup
+
+### Configuring
```

Comment:
Setup and configuring communicates pretty much the same idea, so I would kepe only "Setup" line.

Reply:
I've merged the two sections into only one called "Setting up".